curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/return-a-sorted-array-without-changing-the-original-array.english.md

```diff
@@ -27,7 +27,9 @@ tests:
   - text: <code>nonMutatingSort(globalArray)</code> should return <code>[2, 3, 5, 6, 9]</code>.
     testString: assert(JSON.stringify(nonMutatingSort(globalArray)) === JSON.stringify([2, 3, 5, 6, 9]));
   - text: <code>nonMutatingSort(globalArray)</code> should not be hard coded.
-    testString: assert(!nonMutatingSort.toString().match(/[23569]/g));    
+    testString: assert(!nonMutatingSort.toString().match(/[23569]/g));
+  - text: The function should return a new array, and not mutate the <code>globalArray</code> variable.
```

Comment:
```suggestion
  - text: The function should return a new array, not the array passed to it.
```
This test doesn't check if the array's mutated (other tests cover that, anyway).